test/utils/wasm3_engine.cpp

```diff
@@ -87,8 +88,11 @@ bool Wasm3Engine::instantiate(bytes_view wasm_binary)
         return false;
     }
 
-    auto ret = m3_LinkRawFunction(module, "env", "adler32", "i(ii)", env_adler32);
-    return ret == m3Err_none || ret == m3Err_functionLookupFailed;
+    auto const ret = m3_LinkRawFunction(module, "env", "adler32", "i(ii)", env_adler32);
+    if (ret != m3Err_none && ret != m3Err_functionLookupFailed)
+        return false;
+
+    return m3_RunStart(module) == m3Err_none;
```

Comment:
@gumb0 can a start function access imports? [This](https://webassembly.github.io/spec/core/syntax/modules.html#syntax-start) is not making it clear.

Reply:
It can, for example https://github.com/WebAssembly/spec/blob/ce89050a45fb47e419900eeaca61092b58bde35e/test/core/start.wast#L82

Reply:
Added #742 which tests this.